pinot-core/src/test/java/org/apache/pinot/queries/FluentQueryTest.java

```diff
@@ -506,6 +507,49 @@ public QueryExecuted thenResultIs(String... tableText) {
       return this;
     }
 
+    /**
+     * Asserts that the result of the query as string is the same as given "table".
+     *
+     * The table is a text table. The first row is the header, and the rest of the rows are
+     * the data. Each column must be separated by pipes ({@code |}).
+     * The header is a list of column names and types, e.g. {@code "column1[INT] | column2[STRING]"}.
+     * The data is a list of rows, each row is a list of values separated by pipes, ending with '\n'.
+     *
+     * Contrary to {@link #thenResultIs(String...)}, this method compares input table and query result as strings.
```

Comment:
What is the reason to do that? What is the difference with `thenResultIs`?  Null handling?

Reply:
The real issues I had with thenResultIs(String...) are:
- it requires passing header with types (but doesn't mention column names)
- it produces error message that points to row|col but doesn't show full result (of course that might be problematic if strings are really long)

With this version when assertion fails there is always expected vs actual value, and that actual value can be easily copied back into assertion if it turns out to be correct.